README.md

```diff
@@ -13,13 +13,13 @@ An unofficial, open source Android client for reddit.
 Features
 --------
 
-* Free and open-source Software - no ads/tracking
+* Copylefted libre software, licensed GPLv3+. Use, inspect, change and share at will; with all.
 * Lightweight and fast
 * Swipe posts left and right to perform customizable actions, such as upvote/downvote, or save/hide
-* Advanced cache management - automatically stores past versions of posts and comments
+* Advanced cache management—automatically stores past versions of posts and comments
 * Support for multiple accounts
-* Two-column tablet mode (can be used on your phone, if it’s big enough)
-* Image and comment precaching (optional: always, never, or Wi-Fi only)
+* Two-column tablet mode (can be used on your phone if it’s big enough)
```

Comment:
Why not call it "Two-pane big screen mode" I think it said pane in the strings, but columns is probably better.